mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -1,11 +1,14 @@
 CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
 	FIELD field_21493 IDENTITY Lnet/minecraft/class_1158;
-	FIELD field_21585 a F
+	FIELD field_21582 a F
+	FIELD field_21583 b F
+	FIELD field_21584 c F
+	FIELD field_21585 d F
```

Comment:
No. No. No. `a` is like the whole component while `b` `c` `d` are imaginary components. It is
`a + bi + cj + dk`, not `ai + bj + ck + d` mathematically